CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Command substitutions now have access to the terminal, allowing tools like `fzf` to work in them (#1362, #3922).
 - `bg`s argument parsing has been reworked. It now fails for invalid arguments but allows non-existent jobs (#3909).
 - `read` now supports the `--silent` flag to hide the characters typed (#838).
+- Added completions for `terraform`.
```

Comment:
Note: Usually before a release we'll go through and add all added completions, since it's easier to add that line once, especially when there are multiple PRs that add completions in flight at the same time (merge conflicts).

There's a magical git incarnation for it (as always) - `git diff --name-only --diff-filter=A 2.5.0 share/completions`.